### ISSUE TITLE
[1.15.2] Crash when moving around generating the world on a server, maybe on SP aswell.

### ISSUE THREAD

**Issue by Gbergz** [MysticMods/Druidcraft#41]
### General Information

**Describe the bug:**
Title.

**To Reproduce:**
1. Create a world
2. Move around generating
3. Crash.

**Expected behavior:**
Not crashing.

***

**Mystic Mods Versions**
- DruidCraft version: 1.15-0.4.0

**Other Versions:**
- Forge: 31.2.9
- Minecraft: 1.15.2
- Modpack (if available): Unreleased pack. Not yet available.

***

### Logging Information

**Crash Report:**
https://gist.github.com/Gbergz/f41228b586a0fe8f11dc31db4b895f32

**Comment by VulpTheHorseDog** [comment]
Yup! It's the generating of the darkwood trees. It's been fixed and it should be shipped out with a bunch of other vital fixes very shortly!
